src/nvim/normal.c

```diff
@@ -2019,6 +2019,9 @@ do_mouse (
         if ((State & REPLACE_FLAG) && !yank_register_mline(regname))
           insert_reg(regname, true);
         else {
+          if (regname == 0) {
```

Comment:
any reason to put it here and not before `if ((State & REPLACE_FLAG)...` as it was in `vim` ?